src/20240209_AaveV3Ethereum_TreasuryManagementGSMFundingRWAStrategyPreparationsPart2/TreasuryManagementGSMFundingRWAStrategyPreparationsPart2.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Treasury Management - GSM Funding & RWA Strategy Preparations (Part 2)"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-treasury-management-gsm-funding-rwa-strategy-preparations/16128"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xb39537e468eef8c212c67a539cdc6d802cd857f186a4f66aefd44faaadd6ba19"
+---
+
+## Simple Summary
+
+In preparation for launching the GHO Stability Module (GSM) and the $1M RWA strategy with Centirfuge, this publication seeks to make available the necessary amount of USDC, USDT and DAI on Ethereum.
+
+## Motivation
```

Comment:
GSM was already deployed [here](https://vote.onaave.com/proposal/?proposalId=14&ipfsHash=0x36664a88ec779d778e529d1175762ae8b062bcedc553c07936e158255ad31700) would update the motivation with a more explicit explanation of second phase maybe. 
Provably simple summary also needs an update

Reply:
Replace "launching" with "potentially funding".